packages/server/tests/acceptance/index.spec.ts

```diff
@@ -60,107 +60,112 @@ let startOperatorBalance: Hbar;
+    logger.info(`E2E_RELAY_HOST: ${process.env.E2E_RELAY_HOST}`);
+
+    if (USE_LOCAL_NODE === 'true') {
+      runLocalHederaNetwork();
     }
 
-    function runLocalHederaNetwork() {
-        // set env variables for docker images until local-node is updated
-        process.env['NETWORK_NODE_IMAGE_TAG'] = '0.41.0-alpha.3';
-        process.env['HAVEGED_IMAGE_TAG'] = '0.41.0-alpha.3';
-        process.env['MIRROR_IMAGE_TAG'] = '0.86.0-beta1';
+    if (global.relayIsLocal) {
+      runLocalRelay();
+    }
 
-        console.log(`Docker container versions, services: ${process.env['NETWORK_NODE_IMAGE_TAG']}, mirror: ${process.env['MIRROR_IMAGE_TAG']}`);
+    // cache start balance
+    startOperatorBalance = await global.servicesNode.getOperatorBalance();
+  });
 
-        console.log('Installing local node...');
-        shell.exec(`npm install @hashgraph/hedera-local -g`);
+  after(async function () {
+    const endOperatorBalance = await global.servicesNode.getOperatorBalance();
+    const cost = startOperatorBalance.toTinybars().subtract(endOperatorBalance.toTinybars());
+    logger.info(`Acceptance Tests spent ${Hbar.fromTinybars(cost)}`);
 
-        console.log('Starting local node...');
-        shell.exec(`hedera start -d`);
-        console.log('Hedera Hashgraph local node env started');
+    if (USE_LOCAL_NODE === 'true') {
+      // stop local-node
+      logger.info('Shutdown local node');
+      shell.exec('hedera stop');
     }
 
-    function runLocalRelay() {
-        // start local relay, stop relay instance in local
-        shell.exec('docker stop json-rpc-relay');
-        logger.info(`Start relay on port ${constants}`);
-        relayServer = app.listen({ port: constants.RELAY_PORT });
+    //stop relay
+    logger.info('Stop relay');
+    if (relayServer !== undefined) {
+      relayServer.close();
+    }
 
-        if (process.env.TEST_WS_SERVER === 'true') {
-            logger.info(`Start ws-server on port ${constants.WEB_SOCKET_PORT}`);
-            global.socketServer = wsApp.listen({ port: constants.WEB_SOCKET_PORT });
-        }
+    if (process.env.TEST_WS_SERVER === 'true' && socketServer !== undefined) {
+      socketServer.close();
     }
+  });
+
+  describe('Acceptance tests', async () => {
+    fs.readdirSync(path.resolve(__dirname, './')).forEach((file) => {
+      if (fs.statSync(path.resolve(__dirname, file)).isDirectory()) {
+        fs.readdirSync(path.resolve(__dirname, file)).forEach((subFile) => {
+          loadTest(`${file}/${subFile}`);
+        });
+      } else {
+        loadTest(file);
+      }
+    });
+  });
 
+  function loadTest(testFile) {
+    if (testFile !== 'index.spec.ts' && testFile.endsWith('.spec.ts')) {
+      require(`./${testFile}`);
+    }
+  }
+
+  function runLocalHederaNetwork() {
+    // set env variables for docker images until local-node is updated
+    process.env['NETWORK_NODE_IMAGE_TAG'] = '0.41.0-alpha.3';
```

Comment:
New versions are available 
```suggestion
    process.env['NETWORK_NODE_IMAGE_TAG'] = '0.42.0-alpha.2';
```
or
```suggestion
    process.env['NETWORK_NODE_IMAGE_TAG'] = '0.41.0';
```